ui/index.html

```diff
@@ -28,12 +29,24 @@
       </div>
       <div id="seedContainer" class="container" style="display:none">
          Enter random seed:<br />
-         <input type="text" id="seedTextBox" />
+         <input type="text" id="seedTextBox" maxLength="10" onkeypress="limitKeypress(event,this.value,maxLength)" />
          <button id="seedSaveButton">Save</button>
       </div>
       <div id="listOfBallotsToPull" class="container" style="display:none">
       </div>
       <div id="ballotEntriesContainer" class="outerContainer"></div>
+      <!-- Error box div to display reportError function when an error occurs -->
+      <div id="errorBox" class="modal" data-backdrop="static" data-keyboard="false">
+         <div class="modal-content">
+            <div class="modal-header">
+               <span class="close" id="modalclose">x</span>
+               <h4> Error Message</h4>
+            </div>
+            <div class="modal-body">
+               <h5 id="Message"></h5>
```

Comment:
Could use a more-specific id, like maybe `"errorMessage"`

Reply:
Updated it to errorMessage